apps/crn-server/test/helpers/teardown.ts

```diff
@@ -23,11 +23,14 @@ export const teardownHelper = (
   }
 
   return async () => {
-    try {
-      for (const record of toDelete) {
+    for (const record of toDelete) {
+      try {
         await record.client.delete(record.id);
+      } catch (err) {
+        // eslint-disable-next-line no-console
+        console.log(`Fail to delete entry ${record.id}`);
```

Comment:
Why does the deletion ever fail?

Do we want to also log the error message to give some hope of debugging it?

Reply:
Because I'm deleting the events while I'm running the tests. So when it comes to the teardown some entries don't exist anymore, so it fails to delete.

I'm deleting the events while running the tests so the reminders don't conflict with each other. And I'm not running teardown after each test so I don't need to re-create teams and user every time.

Reply:
About the log to debug it, I'm not really sure if it will become very polluted because it will fail for every event deleted during the test, do you think it's better to add it? :)

Reply:
Well, personally if the failures are routine and expected then I'd probably not log _anything_.

It might be worth checking the error and silently swallowing "Not Found" errors, and logging some useful output if there's any other non-expected errors.